sys/include/net/ng_lomac.h

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (C) 2015 Martine Lenders <mlenders@inf.fu-berlin.de>
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser General
+ * Public License v2.1. See the file LICENSE in the top level directory for
+ * more details.
+ */
+
+/**
+ * @defgroup    sys_net_ng_lomac    Loopback MAC layer
+ * @ingroup     sys_net
+ * @{
+ *
+ * @file
+ * @brief       Interface definition for the loopback MAC layer
+ *
+ * @details     This MAC layer triggers a receive command everytime a
+ *              packet is send to it.
+ *
+ * @author      Martine Lenders <mlenders@inf.fu-berlin.de>
+ */
+
+
+#ifndef NG_LOMAC_H_
+#define NG_LOMAC_H_
+
+#include "kernel_types.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief   Default message queue size for loopback MAC layer.
+ */
+#ifndef NG_LOMAC_MSG_QUEUE_SIZE
+#define NG_LOMAC_MSG_QUEUE_SIZE (8U)
+#endif
+
+/**
+ * @brief   The PID to the loopback MAC layer thread.
+ */
+extern kernel_pid_t lomac_pid;
```

Comment:
@LudwigOrtmann as @cgundogan stated: the MTU can be anything from 1280 to `UINT16_MAX` and is in practice mostly constraint by the underlying MAC layer (for Ethernet I often see 1500 e. g.). The loopback has no such constraints, because it just returns what is given to it. So why constrain it (and following from that adding more work overhead to higher layers) in the first place?